audio_capture/src/audio_capture.cpp

```diff
@@ -203,7 +201,7 @@ namespace audio_transport
         gchar *debug;
 
         gst_message_parse_error(message, &err, &debug);
-        ROS_ERROR_STREAM("gstreamer: " << err->message);
+        // RCLCPP_ERROR_STREAM(this->get_logger(), "gstreamer: " << err->message);
```

Comment:
is it commented for a reason?

Reply:
Yes, since it uses the node instance in order to get the logger (modification from ROS1, then you would just log stuff into due to the existance of a roscore) but the method is static it gives error. We could modify the method but I wasn't sure if it was the right call.

Reply:
I think that it is important to log error here, if you need the instance to get the logger AND it isn't being used out of an instance then remove the static keyword and make it part of the class.

Reply:
The method `g_signal_connect` can't receive a class method because of the implicit first argument `this`. I modified the way the error logging was being made by creating a logger that is not part of the node.